### DISCUSSION TITLE
Server sided mod?

### DISCUSSION THREAD

**Discussion by ivjvon** [MrHydralisk/Lethal-Company-Enhanced-Radar-Booster#8] [Q&A] [answered]
Can't join public lobbies.

**Comment by MrHydralisk** [comment, marked as answer]
It have features that client sided and the ones that require all players to have mod, you can disable some in config. In mod description:

> Client/Server
> 
> - Range increase is fully client-sided and can be used anywhere.
> - Teleportation will require all clients to have it installed to it getting desync. It won't cause errors, if installed only on one client, but you will see Radar Booster being teleported and interacted, but others will see it still being on same place. Won't cause errors.
> - Remote scrap is client-sided, but will make it work for everyone in lobby even without being host.

After Lethal Company v47 it now require same networking objects between client and host, so Teleportation is required by all players in lobby.
